forest_height.R

```diff
@@ -0,0 +1,25 @@
+# Function to decide height of forest plots based on the number of studies
+# and the output format
+# 
+# param studies
+# A number (an int). The number of studies to be plotted.
+# param tab 
+# A string representing the way the plot height is measured
+# "pixel" for png files or plots rendered in the app 
+# "inch" for pdf files
+#
+# return 
+# A number 
+# Height in pixels for png plots
+# Height in inches for pdf plots
+#
+# author
+# NVB
+
+forest_height <- function(studies, measure){
```

Comment:
This might be wider convention that I have not yet come across, but I think that the function could be named in a way that makes it more clear what it is doing, eg "calculate_forest_height"

Reply:
Function names have been changed.